components/wifi_provisioning/src/manager.c

```diff
@@ -961,6 +971,7 @@ esp_err_t wifi_prov_mgr_wifi_scan_start(bool blocking, bool passive,
     }
 
     ESP_LOGD(TAG, "Scan started");
+    prov_ctx->prov_state = WIFI_PROV_STATE_STARTED;
```

Comment:
I would not recommend changing the provisioning SM state as part of scan request here. We can dedicated handler for this, which can be invoked from provisioning entity side. We will discuss this internally and provide update here.

Reply:
well actually this line is a bugfix, otherwise you are able to enter the same function a second time but nobody will change the state of the provisioning to "started".
This will cause the "Scan started" print to be shown when the provisioning SM is actually in a different state.